components/fxa-client/src/jwcrypto.rs

```diff
@@ -0,0 +1,261 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I'm guessing the reason this module is still in `fxa-client` as opposed to `rc_crypto` is because it's only used here right? Seems okay to me, although it doesn't seem `fxa-client`y anymore 😅

Reply:
Yeah that's a good point, alternatively we could create a `jwcrypto` crate in support/.